geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala

```diff
@@ -185,7 +187,8 @@ object AccumuloDataStoreParams {
   val recordThreadsParam   = new Param("recordThreads", classOf[Integer], "The number of threads to use for record retrieval", false, 10)
   val writeThreadsParam    = new Param("writeThreads", classOf[Integer], "The number of threads to use for writing records", false, 10)
   val looseBBoxParam       = new Param("looseBoundingBox", classOf[java.lang.Boolean], "Use loose bounding boxes - queries will be faster but may return extraneous results", false, true)
-  val statsParam           = new Param("collectStats", classOf[java.lang.Boolean], "Toggle collection of statistics", false, true)
```

Comment:
Ok, this'll break folks as if they upgrade their GM Accumulo GeoServer plugin; they'll have to re-register datastores, etc., since the xml for the original will have "collectStats". 

Not sure if this should block the change or not.  @jahhulbert-ccri thoughts?